R-package/R/xgb.DMatrix.R

```diff
@@ -9,12 +9,13 @@
 #' method (`tree_method = "hist"`, which is the default algorithm), but is not usable for the
 #' sorted-indices method (`tree_method = "exact"`), nor for the approximate method
 #' (`tree_method = "approx"`).
+#'
 #' @param data Data from which to create a DMatrix, which can then be used for fitting models or
 #' for getting predictions out of a fitted model.
 #'
 #' Supported input types are as follows:\itemize{
 #' \item `matrix` objects, with types `numeric`, `integer`, or `logical`.
-#' \item `data.frame` objects, with columns of types `numeric`, `integer`, `logical`, or `factor`.
+#' \item `data.frame` objects, with columns of types `numeric`, `integer`, `logical`, or `factor`}
```

Comment:
Looks like we have a mixture of markdown ("-" for bullet points) and roxygen ("\itemize" for list and "\item" for each point).

Should ideally use only one format for a given function doc.

Reply:
Will convert them

Reply:
Done.